src/constants/autocompletes/countingChannels.ts

```diff
@@ -9,6 +9,13 @@ const autocomplete: Autocomplete = {
     const search = String(query);
 
     const countingChannels = Array.from(document.channels.entries());
+    if (!countingChannels.length) {
```

Comment:
maybe change to == 0, as practically its the same, but is syntactically more clear what you are trying